src/late/ipfs-on-path/scripts/install.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs-extra')
+const which = require('which')
+const { backup } = require('../../../utils/scripts/backup')
+const getArg = require('../../../utils/scripts/args')
+const { SOURCE_SCRIPT, DEST_SCRIPT } = require('./consts')
+
+// TODO: be careful of file permissions when fs.something()
+
+let exists = false
+
+try {
+  fs.lstatSync(DEST_SCRIPT)
+  exists = true
+  console.log(`${DEST_SCRIPT} already exists`)
```

Comment:
logger

Reply:
Oh i see, this script is excuted seperately...

Reply:
😃 Yes, what I said above!